Tensile/Source/lib/source/ContractionProblem.cpp

```diff
@@ -156,8 +156,7 @@ namespace Tensile
 
         batchIndices.push_back({2,2,2,2});
 
-        if(beta != 0.0)
-            c = d;
+        c = d;
```

Comment:
Should this if be gone?

Reply:
Yes, not having c = d makes it a different problem based on whether beta == 0 which is not exactly the case.